routes/shopifyApiProxy.js

```diff
@@ -6,19 +6,13 @@ module.exports = function shopifyApiProxy(request, response, next) {
   const { query, method, path, body, session } = request;
   const { shop, accessToken } = session;
 
-  const strippedPath = path.split('?')[0].split('.json')[0];
-
-  const inAllowed = ALLOWED_URLS.some(resource => {
-    return strippedPath === resource;
-  });
-
-  if (!inAllowed) {
+  if (!validRequest(path)) {
     return response.status(403).send('Endpoint not in whitelist');
   }
 
   const fetchOptions = {
     method,
-    body,
+    body: body,
```

Comment:
nit: can just do `body,` on this line thanks to the magic of es6

Reply:
ooh right